persistence/module.go

```diff
@@ -64,6 +75,18 @@ func Create(cfg *genesis.Config, genesis *genesis.GenesisState) (modules.Persist
 	return persistenceMod, nil
 }
 
+func InitGenesis(data json.RawMessage) (genesis *types.PersistenceGenesisState, err error) {
+	genesis = new(types.PersistenceGenesisState)
+	err = json.Unmarshal(data, genesis)
+	return
+}
+
+func InitConfig(data json.RawMessage) (config *types.PersistenceConfig, err error) {
```

Comment:
As discussed in protocol hour, let's add a unit test for each config initialisation and each genesis initialisation for each module where we do this.